certbot_dns_azure/_internal/dns_azure.py

```diff
@@ -65,6 +87,15 @@ def _validate_credentials(self, credentials):
                                      ' e.g dns_azure_zone1 = DOMAIN:DNS_ZONE_RESOURCE_GROUP_ID'
                                      ''.format(credentials.confobj.filename))
 
+        # Azure Environment
+        environment = credentials.conf('environment')
+
+        if environment != None:
```

Comment:
`if environment` should suffice, as None would evaluate to false anyway.

Reply:
Thanks, Updated